openlibrary/plugins/openlibrary/js/graphs/plot.js

```diff
@@ -226,7 +227,7 @@ export function loadGraph(id, options = {}, tooltip_message = '', color = null)
         if (tooltip_message) {
             return plot_tooltip_graph($(node), data, tooltip_message, color);
         } else {
-            return $.plot($(node), data, options);
+            return $.plot($(node), [{data: data}], options);
```

Comment:
The `books-added-per-day` graph appears to be the only graph that reaches this `else` block of code.  Related file: https://github.com/internetarchive/openlibrary/blob/master/openlibrary/plugins/openlibrary/js/graphs/index.js